modules/fee-market/src/lib.rs

```diff
@@ -346,7 +373,16 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	/// - The enrolled relayer wants to update fee or order capacity.
 	/// - The enrolled relayer wants to cancel enrollment.
 	/// - The order didn't confirm in-time, slash occurred.
-	pub(crate) fn update_market() {
+	pub(crate) fn update_market<F>(f: F, has_event: Option<Event<T, I>>) -> DispatchResult
```

Comment:
I mean we can change this to `DispatchResult`.
Keep `DispatchResultWithPostInfo` for the extrinsic call.

Reply:
update?

Reply:
According to [DispatchResult Doc](https://crates.parity.io/frame_support/dispatch/type.DispatchResult.html):
> Should be used whenever the PostDispatchInfo does not need to be overwritten.

We don't need to overwrite the post-dispatch info in fee market cases. So I think `DispatchResult` is okay here.